clickhouse_sqlalchemy/drivers/native/connector.py

```diff
@@ -11,14 +11,7 @@
 threadsafety = 2
 # Python extended format codes, e.g. ...WHERE name=%(name)s
 paramstyle = 'pyformat'
-
-
-class Error(Exception):
-    """
-    Exception that is the base class of all other error exceptions.
-    You can use this to catch all errors with one single except statement.
-    """
-    pass
+Error = DatabaseException
```

Comment:
Why do we change Exception to DatabaseException? Ordinary exceptions can be thrown as well.

Reply:
I'm just a little confused about what DatabaseException does.
If the ordinary exceptions is also thrown, then it should be `Exception` or `clickhouse_driver.Error`?